src/lib/i18n/dictionaries/dictionary.json

```diff
@@ -3717,7 +3717,10 @@
   "crafting.readySoon": "{{name}} will be ready soon...",
   "crafting.expansionSoon": "Your land will be ready soon...",
   "crafting.viewRecipes": "View recipes",
-  "gems.buyReplenish": "Would to like to replenish the shops now for 20 Gems?",
+  "gems.buyReplenish": "Would you like to replenish the shops now for 20 Gems?",
+  "gems.clearStock": "You might want to buy all your stock before restocking",
```

Comment:
I don't think this message is necessary on the modal - or if this information is needed, it might be less confrontational to simply state how the feature works:

<img width="421" alt="1" src="https://github.com/user-attachments/assets/1ed9452b-c84b-4749-86c4-4f9870007219">